docs/components/InputDate/Mobile.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import omit from "lodash/omit";
+import { InputDate } from "@jobber/components-native";
+
+export default {
+  title: "Components/Forms and Inputs/InputDate/Mobile",
+  component: InputDate,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
```

Comment:
Code Tab doesn't work (pretty sure Darryl has a PR open for me to review which fixes this).  Even still, it's import is incorrect (not pulling from coponents-native)

![Screenshot 2023-07-19 at 10 31 12 AM](https://github.com/GetJobber/atlantis/assets/104797704/539f6fa7-fbab-48c2-ab2d-f3eb44f7fa85)